automatic/ruby/ruby.nuspec

```
@@ -34,13 +26,9 @@ Example: `choco install ruby --params "/NoPath /InstallDir:C:\your\install\path"
     <releaseNotes>https://www.ruby-lang.org/en/downloads/releases/</releaseNotes>
     <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@aad7c15bfbec43c3716f8a82bc3af22e1a55579d/icons/ruby.svg</iconUrl>
     <dependencies>
-      <dependency id="chocolatey-core.extension" version="1.3.3" />
-      <dependency id="chocolatey" version="0.10.5" />
+        <dependency id="ruby.install" version="[3.1.0.1]" />
     </dependencies>
   </metadata>
-  <files>
-    <file src="tools\**" target="tools" />
-    <file src="legal\**" target="legal" />
-  </files>
+  <files></files>
```

To follow what is done for other packages.

```suggestion
  <files />
```